src/BrowserKitDriver.php

```diff
@@ -413,7 +413,24 @@ public function getValue($xpath)
      */
     public function setValue($xpath, $value)
     {
-        $this->getFormField($xpath)->setValue($value);
+        $formField = $this->getFormField($xpath);
+
+        try {
```

Comment:
It can't be.

The code iterates over options and if title matches it returns the `value` attribute value.

So if you search `title one` you'll get `''` (empty string) back from `getSelectValuesByOptionTest` method and attempt to select it would select first/last (depending on how BrowserKit uses it) option with matching `value` attribute value. So there is no assurance that `title one` option would be selected all the time.

Best to try with 3 options and search for one in the middle.